src/Components/PageSelection/PageSelection.js

```diff
@@ -14,15 +14,19 @@ const PageSelection = (props) => {
   return (
     <Fragment>
       {selectedPage === Pages.HOME && (
-        <Home
-          allProducts={props.allProducts}
-          changePage={setSelectedPage}
-          setStaticHeader={props.setStaticHeader}
-          updateProduct={props.updateProduct}
-          removeProduct={props.removeProduct}
-          setCurrentProduct={props.setCurrentProduct}
-          increaseLikeCount={props.increaseLikeCount}
-        />
+        <Fragment>
```

Comment:
We would use `Fragment` here if we have multiple elements. Just a side note, changes are not necessary.

Reply:
Right, I used to have a couple of things here at some point in development and missed to remove the Fragment in the end.

Reply:
Thanks
f719f38